tests/test_permissions.py

```diff
@@ -56,6 +57,17 @@ def test_is_authenticated(db, gql_client: GraphQLTestClient):
         }
 
 
+@pytest.mark.django_db(transaction=True)
+async def test_channels_authenticated(
```

Comment:
Would be nice to have tests for both authenticated and non-authenticated (which would fail) for this